bucket/neochat-nightly.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "346",
+    "description": "A client for Matrix, the decentralized communication protocol",
+    "homepage": "https://apps.kde.org/neochat/",
+    "license": "GPL-3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://binary-factory.kde.org/job/NeoChat_Nightly_win64/lastSuccessfulBuild/artifact/neochat-master-346-windows-msvc2019_64-cl.7z",
+            "hash": "6c29b766cf140c95d0bd7780ea99d8bedab03130846a6324074a84d8ae37cb7a"
+        }
+    },
+    "bin": [
+        [
+            "bin\\neochat.exe",
+            "neochat-nightly"
```

Comment:
Hmm, what if in the future a stable neochat is released? For example, I made the shim for Okular Nightly `okular-nightly` so as not to conflict with the stable Okular in the extras bucket.

https://github.com/ScoopInstaller/Versions/blob/18a4582f1345affa3b8f8afe884d494d3ed97f74/bucket/okular-nightly.json#L12-L16

Reply:
Why not reset them? Do you have the need to use stable version and nightly version?

Reply:
Personally no, but I thought the whole point of the versions bucket was so that you could install old and new / beta and stable software side by side? I think people prefer having them side by side rather than resetting them each time they need one or the other.

Reply:
Check all the manifests in `Versions` except your one, they all overwrite original shims and are switched by `scoop reset`. The side-by-side installation doesn't mean you have two different shims, but just reset without problem.

Reply:
Oh okay. So all the Python37, Python38, Python39 etc. installations having different shims for side by side use are a special case then?

Reply:
https://github.com/ScoopInstaller/Versions/blob/c76043d75c36a72745f13b972878f24edb737cf8/bucket/python39.json#L34-L44
https://github.com/ScoopInstaller/Versions/blob/c76043d75c36a72745f13b972878f24edb737cf8/bucket/python38.json#L34-L44
https://github.com/ScoopInstaller/Versions/blob/c76043d75c36a72745f13b972878f24edb737cf8/bucket/python37.json#L34-L44

Notice the top two line with `python.exe` and `pythonw.exe`? They are for reset.

Reply:
Okay then, I'll make your change.